### DISCUSSION TITLE
Subdir name not stripped from webradio URL if URL is rtsp

### DISCUSSION THREAD

**Discussion by vale-max** [rern/rAudio#941] [Fixed Issues]
If you put a webradio whose URL is rtsp in a subfolder then the directory name becomes part of the URL and the radio is not playable

e.g. 
Create subfolder test
in subfolder test add radio with URL rtsp://hello_world and save it
try editing the radio, the url will be displayed as test/rtsp://hello_world and the radio can not be added to the playlist

**Comment by rern** [comment]
To be fixed by next update.
